hazelcast/src/main/java/com/hazelcast/config/MapConfig.java

```diff
@@ -796,6 +855,7 @@ public String toString() {
                 + ", mapAttributeConfigs=" + mapAttributeConfigs
                 + ", quorumName=" + quorumName
                 + ", queryCacheConfigs=" + queryCacheConfigs
+                + ", cacheDeserializedValues=" + cacheDeserializedValues
                 + '}';
```

Comment:
I meant that the `cacheDeserializedValues` it's used in equals and not used in the hashCode method. In theory it's fine but a bit unusual, thus my question. The only reason it's like this I could figure out was the performance of hashCode. On the other hand it's just a boolean.